File: src/main/java/com/github/eendroroy/sdk/openweather/Unit.java
package com.github.eendroroy.sdk.openweather;

/**
 * @author indrajit
 */

public enum Unit {
    STANDARD("standard"),
    METRIC("metric"),
    IMPERIAL("imperial");

    private final String value;

    Unit(String value) {
        this.value = value;
    }

    @Override
    public String toString(){
        return this.value;
    }

    public String value(){
        return this.value;
    }
}

File: src/main/kotlin/com/github/eendroroy/sdk/openweathermap/Unit.kt
package com.github.eendroroy.sdk.openweathermap

/**
 * @author indrajit
 */
enum class Unit(private val value: String) {
    STANDARD("standard"), METRIC("metric"), IMPERIAL("imperial");

    override fun toString(): String {
        return value
    }

    fun value(): String {
        return value
    }

}